src/templates/navbar-fixed.html

```diff
@@ -54,10 +54,11 @@
             <div class="visible-xs-inline navbar-mobile-xp-container">
                 <ul class="nav navbar-nav navbar-mobile-xp">
                     <li>
-                        {% comment %} # DISABLE MARKS {% endcomment %}
-                        {% comment %} <a href="{% url 'courses:marks' request.user.id %}" {% endcomment %}
-                        <a href="#" title="The Mark Calculations page is currently disabled, sorry!"
-                        class="navbar-text-large-container">XP:
+                        {% url 'courses:marks' request.user.id as marks_url %}
```

Comment:
> #What happens if the marks calculation page is disabled? Why `marks_url` (and what is that) vs `{% url 'courses:marks' request.user.id %}`? Sorry I don't recall much of this code and need more context and details specific to your decisions.
> 
> This is the detail that should be in your PR description (at least it would really help me review and accept PRs quicker) Thanks!

I copied most of the code from the desktop version of the navbar. Using `marks_url` just makes the code underneath a bit more simple. I added a more detailed description in my PR.